src/index.html

```diff
@@ -7,6 +7,41 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="main">
+      <h1>Calendar</h1>
+      <div class="calendar">
```

Comment:
```suggestion
      <div class="calendar calendar--28 calendar--sun">
```
Тут еще должно быть два класса модификатора, которые определяют сколько дней в календаре и какой день первый?